Integrations/QRadar/QRadar.py

```diff
@@ -1091,6 +1214,8 @@ def get_domains_by_id_command():
         demisto.results(get_domains_command())
     elif demisto.command() == 'qradar-get-domain-by-id':
         demisto.results(get_domains_by_id_command())
+    elif demisto.command() == 'qradar-upload-indicators':
+        return_outputs(upload_indicators_command())
```

Comment:
you are not returning anything to this place, please fix it

Reply:
done